### PR TITLE
Pull request to add reviewer.

### PR DESCRIPTION
**Pull request by Jamesw2013**

### CODE REVIEW

File: assignments/arrays.js [hunk 1]
```diff
@@ -63,32 +63,53 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
+    if(inventory[i].id === 33){
+        console.log('Car 33 is a', inventory[i]['car_year'], inventory[i]['car_make'], inventory[i]['car_model'] + '.');
+    }
+}
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory[inventory.length-1];
+console.log(lastCar.car_make + " " + lastCar.car_model);
```
Comment (Mister-Corn) [inline assignments/arrays.js:74]: You can use template literals to construct strings as well. For example, 

```js
console.log(`The car make is ${lastCar.car_make}, and the car model is ${lastCar.car_model}`);
```

File: assignments/arrays.js [hunk 2]
```diff
@@ -63,32 +63,53 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+for (var i=0; i < inventory.length;i++){
```
Comment (Mister-Corn) [inline assignments/arrays.js:79]: IMHO, using `var` in for loops is a risky proposition. Unlike `let` and `const`, `var` is only function-scoped, and will leak out of blocks like if statements and for loops. In this case, everything is fine because you continually reset `i` to zero, but be aware that `i` exists outside of the for loop because you used `var`, and depending on the circumstance or surrounding code, that may lead to unexpected behavior.

TLDR: I strongly suggest you use `const` and `let`.

File: assignments/function-conversion.js [hunk 3]
```diff
@@ -4,22 +4,27 @@
...
 // };
 // myFunction();
-
+let myFunction = ()=> {console.log("Function was invoked!")}
+myFunction();
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
+let anotherFunction = (param) => {return param;}
```
Comment (Mister-Corn) [inline assignments/function-conversion.js:13]: You don't have to write it this way, but keep in back of your mind that if the only thing the function is doing is returning an expression, you'll often see the short implicit return form like this:

```js
let anotherFunction = (param) => param;
```

**Review by Mister-Corn — APPROVED** [review]
## Assignment Objectives 

### Objects

- [x] Complete each challenge presented before moving on to Arrays.

### Arrays

 Use for loops and arrays to solve their problems.

- [x] Utilize the the array `inventory` to complete your challenges
- [x] You are not permitted to use map, reduce, or filter to solve these problems.  Only use a basic for loop.
- [x] Complete each challenge presented before moving on to stretch.

### Arrow Function Syntax

* [x] Convert functions to  Arrow Function Syntax

### Comments

Good job, James! You demonstrated a good grasp of the concepts, and MVP is looking nice. Keep up the good work! :+1:
